core/clients/blobberClient/client.go

```diff
@@ -270,3 +271,27 @@ func CalculateHash(url string, req *blobbergrpc.CalculateHashRequest) ([]byte, e
 
 	return json.Marshal(convert.GetCalculateHashResponseHandler(calculateHashResp))
 }
+
+func RenameObject(url string, req *blobbergrpc.RenameObjectRequest) ([]byte, error) {
+	blobberClient, err := newBlobberGRPCClient(url)
+	if err != nil {
+		return nil, err
+	}
+
+	clientSignature, err := client.Sign(encryption.Hash(req.Allocation))
+	if err != nil {
+		return nil, err
+	}
+
+	grpcCtx := metadata.NewOutgoingContext(context.Background(), metadata.New(map[string]string{
+		blobbercommon.ClientHeader:          client.GetClientID(),
+		blobbercommon.ClientKeyHeader:       client.GetClientPublicKey(),
+		blobbercommon.ClientSignatureHeader: clientSignature,
+	}))
+
+	renameObjectResp, err := blobberClient.RenameObject(grpcCtx, req)
+	if err != nil {
+		return nil, err
+	}
+	return json.Marshal(renameObjectResp)
```

Comment:
Dont we need to convert the object to the necessary type?

Reply:
There are two reasons for that:
- FIrst being that we do not have a response handler for renameObjectResp in "github.com/0chain/blobber/code/go/0chain.net/blobbercore/convert"
   We need to create one there first.
- Second being that we do not actually make any use of the response object as of right now.

So I'm not sure, if I have to create a response handler in the blobber repo, and then after it is updated, use in this one. What do you suggest? @kushthedude

Reply:
@kushthedude I added the renameObjectResponse handler in blobber repo: https://github.com/0chain/blobber/pull/296
. I used that commit hash for the this commit for the blobber package, utilizing the newly created renameObjectResponse handler. Converting it to the necessary type.

Reply:
Everything is already present in the grpc_v2 branch of blobber.

Reply:
@kushthedude I couldn't find the responsehandler for rename object in grpc_v2 branch of the blobber.
It should be here right?
https://github.com/0chain/blobber/blob/grpc_v2/code/go/0chain.net/blobbercore/convert/response_handler.go